full_snapshot.proto

```diff
@@ -39,6 +39,7 @@ message FullSnapshot {
 
   repeated TablespaceReference tablespace_references = 130;
   repeated TablespaceInformation tablespace_informations = 131;
+  CollectorConfig config = 132;
```

Comment:
I think we could number this `13` and place it after `collected_interval_secs`

Reply:
Or, after `collector_errors` and number it 22. Not sure which group fits better.